src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -171,6 +171,20 @@
 				'@gitpod/local-app-api-grpcweb': `${window.location.origin}/static/node_modules/@gitpod/local-app-api-grpcweb/lib/localapp.js`,
 				'@improbable-eng/grpc-web': `${window.location.origin}/static/node_modules/@improbable-eng/grpc-web/dist/grpc-web-client.umd.js`,
 				'@zip.js/zipjs': `${window.location.origin}/static/node_modules/@zip.js/zip.js/dist/zip-no-worker-deflate.min.js`
+			},
+			onError: function (err) {
```

Comment:
Due to a bug in https://github.com/microsoft/vscode-loader , we can't get the address of the original url, so we have to use the moduleID instead

Reply:
because here is no `event` object, so we cannot reuse `onVsCodeWorkbenchError` logic, maybe need refactor